_config.yml

```diff
@@ -12,6 +12,7 @@ description: > # this means to ignore newlines until "baseurl:"
   We do Ruby on Rails and Java applications in Play! Framework or Spring MVC.
 baseurl: "" # the subpath of your site, e.g. /blog
 url: "http://beyondscheme.com" # the base hostname & protocol for your site
+permalink: /:year/:title
```

Comment:
You can remove your local directory `_site` where are old posts compiled with the old permalink pattern.